azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub.py

```diff
@@ -95,6 +101,72 @@ def on_token_update_complete(op, error):
         )
 
 
+class EnsureDesiredPropertiesStage(PipelineStage):
```

Comment:
>EnsureDesiredPropertiesStage [](start = 6, length = 28)

would you like to put some doc here ? like the other stages

Reply:
+1

---
In reply to: [401865205](https://github.com/Azure/azure-iot-sdk-python/pull/522#discussion_r401865205) [](ancestors = 401865205)